determined-ai/determined
# docs: Batch Processing API doc

## Description
Add documentation for the new torch batch processing API



## Test Plan
- make live to ensure docs build
- Click through the docs to ensure it looks as expected


## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
